src/teehr/evaluation/tables.py

```diff
@@ -399,7 +399,9 @@ def to_pandas(self):
         """Return Pandas DataFrame."""
         if self.df is None:
             self._raise_missing_table_error()
-        return self.df.toPandas()
+        df = self.df.toPandas()
+        df.attrs['table_type'] = None
```

Comment:
This is a general comment about assigning the attributes...when we're assigning the attributes in each case/table, it seems we have access to `self` which is the table or metric class object.  And each instance of `self` has a bunch of attributes associated with it that could potentially be useful for the plotting methods.  

For tables (timeseries or locations for example) you have access to the table model through `self.table_model` which contains the schema (or I guess `self.fields()` will return the field names as well). This could allow you to avoid importing the models in the accessor class and provide the actual fields for the `joined_timeseries` table since they are dynamic.  Not sure if it makes sense to include the actual models in the attributes?

I guess it's a little different for the metrics but we have access to the `Metrics` class and could also include info about each requested metric in the `include_metrics` list.  Just kind of brainstorming, hopefully this makes sense!

Reply:
I agree.  @samlamont  I was also thinking about this with respect to the Pandera models which is why I wanted to move them from the loading into the table class (although we didn't get there in the recent refactor).  Probably need to have a little bit higher level plan for what this could look like.  Lets discuss.

Reply:
Per our discussion on 10/23 -- created new DataFrame attribute 'fields' and populate using each tables respective table model. Updated validation method in the accessor class to consider this change.